amalgam/api.py

```diff
@@ -1058,6 +1132,56 @@ def execute_entity_json(
 
         return result
 
+    def execute_entity_json_logged(
+            self,
+            handle: str,
+            label: str,
+            json: str | bytes
+    ) -> ResultWithLog:
```

Comment:
Over indented parameters
```suggestion

    def execute_entity_json_logged(
        self,
        handle: str,
        label: str,
        json: str | bytes
    ) -> ResultWithLog:
```